doc/CONTRIBUTING.md

```diff
@@ -14,3 +14,105 @@ Suggested branches naming convetion: `task_number-task_name`.
 Commit message naming convention: `task_number if implemented this commit will`.
 
 To get task numbers, use: [Trello Card Numbers Plus](https://chrome.google.com/webstore/detail/trello-card-numbers-plus/ncibjlmfhjcjnphnpphgphbflpdpliei/related)
+
+## API documentation
+
+Swagger 3 (OpenAPI 3) is used for API documentation.
+
+### Writing an API documentation
+To create API documentation with OpenAPI you need Spring Annotations.
+In your controller classes, use Spring annotations to define API endpoints and provide additional metadata for the OpenAPI documentation. Some commonly used annotations include:
+- `@RestController: Indicate that the class is a controller for handling REST requests.`
+- `@RequestMapping: Define the base path for the controller's endpoints.`
+- `@GetMapping, @PostMapping, etc.: Define specific HTTP methods and paths for your endpoints.`
+- `@Operation: Provide descriptions for your API operations.`
+- `@Parameter: Describe parameters for your API methods.`
+- `@ApiResponse: Define responses for different scenarios.`
+- `@ApiResponses: Group multiple @ApiResponse annotations.`
+
+Example:
+```java
+@RestController
+@RequestMapping("/api")
+public class UserController {
+
+    @GetMapping("/user/{id}")
+    @Operation(summary = "Get user by ID")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "User found"),
+        @ApiResponse(responseCode = "404", description = "User not found")
+    })
+    public ResponseEntity<User> getUserById(@PathVariable Long id) {
+        // Implementation
+    }
+}
+```
```

Comment:
This looks like an example copied from some external docs. If that's the case, remove this example and provide a link instead.